flink-cdc-connect/flink-cdc-source-connectors/flink-connector-mysql-cdc/src/main/java/org/apache/flink/cdc/connectors/mysql/debezium/task/MySqlSnapshotSplitReadTask.java

```diff
@@ -109,6 +117,21 @@ public MySqlSnapshotSplitReadTask(
         this.snapshotChangeEventSourceMetrics = snapshotChangeEventSourceMetrics;
         this.hooks = hooks;
         this.isBackfillSkipped = isBackfillSkipped;
+
+        // setting column filter
+        String columnIncludeList =
+                sourceConfig.getDbzConfiguration().getString(COLUMN_INCLUDE_LIST);
+        String columnExcludeList =
+                sourceConfig.getDbzConfiguration().getString(COLUMN_EXCLUDE_LIST);
+        if (columnIncludeList != null) {
+            this.columnFilter =
+                    Tables.ColumnNameFilterFactory.createIncludeListFilter(
+                            columnIncludeList, ColumnFilterMode.CATALOG);
+        } else {
+            this.columnFilter =
+                    Tables.ColumnNameFilterFactory.createExcludeListFilter(
+                            columnExcludeList, ColumnFilterMode.CATALOG);
+        }
```

Comment:
Only one of `COLUMN_INCLUDE_LIST` and `COLUMN_EXCLUDE_LIST` will take effect in snapshot phase. 
But both of them will pass as the debezium setting in the incremental phase. 
Will this setting cause the different behavior between snapshot phase and incremental phase?

Reply:
This will only take effect in the snapshot phase, because MySQL's binlog cannot be configured to capture partial columns. In the incremental phase, projection can be used to query partial columns.